roles/BCPC-Hadoop-Worker.json

```diff
@@ -20,7 +20,8 @@
     "recipe[bcpc-hadoop::oozie_client]",
     "recipe[bcpc::zabbix-work]",
     "recipe[bcpc::diamond]",
-    "recipe[bcpc_jmxtrans]"
+    "recipe[bcpc_jmxtrans]",
+    "recipe[bach_spark::default]"
```

Comment:
Hive and PIG are installed separately. Hive is role and PIG installs as a separate recipe. We can introduce a new role for Hadoop Eco System components.